external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -72,8 +72,10 @@
     private transient boolean initialized;                              // Flag indicating that the spout is still undergoing initialization process.
     // Initialization is only complete after the first call to  KafkaSpoutConsumerRebalanceListener.onPartitionsAssigned()
 
+    private KafkaSpoutStream kafkaSpoutStream;
```

Comment:
KafkaSpoutStreams is a collection of KafkaSpoutStream because you can have multiple topics for the kafkaconsumer.subscribe(List<String topics... )

That is not the case of kafkaconsumer.subscribe(Pattern ).  You can only have one pattern per Spout, so it doesn't make sense to put in Streams instead of Stream